cachito/workers/cleanup_job.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+import logging
+from datetime import datetime, timedelta
+
+import requests
+
+from cachito.workers.requests import get_requests_session
+from cachito.workers.config import get_worker_config
+from cachito.errors import CachitoError
+
+log = logging.getLogger(__name__)
+
+auth_session = get_requests_session(auth=True)
+session = get_requests_session()
+config = get_worker_config()
+
+# state, state_reason and payload values for marking a request in Cachito API
+state = 'stale'
+state_reason = 'The request has expired'
+payload = {'state': state, 'state_reason': state_reason}
```

Comment:
This is only used in the `mark_as_stale` function. I'd prefer if this wasn't global to the `cleanup_job` module.

Reply:
I made it global so that it doesn't get initialized on every call to that function (which will happen for every stale request id).

Reply:
Alright. I don't have a strong preference then.  :)